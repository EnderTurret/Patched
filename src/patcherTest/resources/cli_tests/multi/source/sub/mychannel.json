{
  "Content": {
    "subscribers": 2,
    "videos": [
      {
        "name": "How to",
        "likes": 1
      }
    ]
  }
}