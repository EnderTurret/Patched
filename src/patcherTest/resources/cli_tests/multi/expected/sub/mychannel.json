{
  "Content": {
    "subscribers": 999531,
    "videos": [
      {
        "name": "How to",
        "likes": 1
      },
      {
        "name": "Integer Unboxing",
        "likes": 312
      }
    ]
  }
}